caracara/modules/hosts/hosts.py

```diff
@@ -114,25 +126,50 @@ def _perform_action(self, action_name: str, device_ids: List[str]) -> Dict:
         return returned
 
     @filter_string
-    def describe_devices(self, filters: FalconFilter or str = None) -> Dict[str, Dict]:
+    def describe_devices(
+        self,
+        filters: FalconFilter or str = None,
+        online_state: OnlineState or str = None,
```

Comment:
It's probably more accurate to make this an Optiona[].
```suggestion
        online_state: Optional[Union[OnlineState, str]],
```